doc/project/checklist.md

```diff
@@ -262,6 +262,10 @@ Set up FPV regression by adding the module to `hw/formal/fpv_all` script.
 
 Sub-modules that are pre-verified with their own testbenches have already reached V1 or higher stage.
 
+### TB_LINT_PASSING
```

Comment:
```suggestion
### LINT_SETUP
```

Seems like this could apply to both, DV as well as FPV testbenches? Any thoughts on whether to add FPV specific hjson as well?

I think we should add V2 checklist also that says `LINT_CLEAN`. WDYT?

Reply:
Thanks for the comments Sri. We added a FPV hjson already [here](https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson).
The reason I did not use `LINT_SETUP` because it is already used in design, I think later when we reference it, it might cause some issue if they are the same name. 
I think `V1` has to be lint_clean because we add the Lint checking in `CI`. If there is any warnings or errors, the CI will fail.
Maybe we can name it `LINT_CLEAN` for V1?

Reply:
I would be in favor of mirroring what we do on the design side. i.e., I think we should have something like
`LINT_SETUP` for V1, `LINT_PASS` for V2 and `LINT_COMPLETE` for V3. granted, if we're just using Verible for the time being, we'll always be lint clean from the beginning since CI does not let you check in anything. but that might change in the future. also, Verible supports lint waivers, and DV code typically has a higher chance of needing waivers than pure RTL code. these waivers need to be signed off upon in V2/V3 (see description of D* items).

In order to make the checklist item names unique, we could just prepend it with `DV_`, e.g. `DV_LINT_SETUP` etc... WDYT?

Reply:
oh and I agree that these items should apply to both DV and FPV since, as @cindychip mentioned, we now also added a lint regression for FPV testbenches.

Reply:
Agree with @msfschaffner, but the current `TB_` prefix seems better.

Reply:
I am fine with `TB_` as well.

Reply:
Thank you for all the feedback and suggestions. I will add a `TB_LINT_SETUP`, `TB_LINT_PASS`, and `TB_LINT_COMPLETE` as recommended.